### DISCUSSION TITLE
Stdnt Dev Pack - Approved Sep 30 - Still

### DISCUSSION THREAD

**Discussion by aziggy-00** [community/community#140505] [GitHub Education]
### Hi everyone,

Hello I have been approved for the student developer pack and the FAQ says to reach out here if it has been 5 days without receiving access. It says "Your request was approved. [Your benefits will be available soon.]"

**Comment by davevad93** [comment]
Hi @aziggy-00 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @aziggy-00 was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
